### DISCUSSION TITLE
Add more notepages

### DISCUSSION THREAD

**Discussion by l4v4andul4** [kudrykv/latex-yearly-planner#121] [Q&A]
Hello,

thanks for this excellent planner. I use it as my daily driver on my Remarkable2.

I already modified some templates to fit better to my use case. But i only know LaTeX and have no idea, how to deal with golang.

I wish to add some extra pages to the note section. So the idea is, that the notes index stays the same, but that every number in the index points not only to one page with lines but to the first page of a bunch of four our five note pages.  
Is this possible?

I use a slightly modified version of the rm2.breadcrumb.lined.default.dailycal.

I presume, that i have to change something in the notes_indexed.go.

Perhaps this question is similar to [this one](https://github.com/kudrykv/latex-yearly-planner/discussions/80)? But i didn´t exactly understand the solution of that question.

Every help is welcome!
Thank you!

**Comment by kudrykv** [comment]
Hello,

Thank you for your interest in planner.
Yes, the post you've referred is the "solution". [The code mentioned there](https://github.com/kudrykv/latex-yearly-planner/discussions/80#discussioncomment-5943397) is likely a "solution" (it looks like one, though, I didn't try it out).

The idea there is to modify `DailyStuff` function in such way, so it receives the desired number of pages as a third parameter, and multiply note pages in a loop

**Reply by l4v4andul4** [reply]
Thank you for your quick response. So I am on the right way.
As soon, as I have the time, I will try to adapt this answer to the notes_indexed-file.
